RetailCoder.VBE/Refactorings/EncapsulateField/EncapsulateFieldPresenter.cs

```diff
@@ -46,50 +47,51 @@ public EncapsulateFieldModel Show()
 
```

Comment:
I think starting a line 38 we have a new method - there's something that doesn't feel right about that `if..else` tree here. Whether the viewmodel `CanHaveLet` or `CanHaveSet` would be much clearer if it were one statement - perhaps with some functions for proper abstraction, but I'm pretty sure I don't like that `if..else` tree.